docs/readmes/architecture/agw_overview.md

```diff
@@ -0,0 +1,59 @@
+---
+id: agw_overview
+title: AGW Overview
+hide_title: true
+---
+
+# Access Gateway Overview
+
+The main service within the access gateway (AGW) is Magmad, which brings up all the services hosted within the AGW. The other major services and components hosted within the AGW are: the MME, S-PGW, Health Checker, Mobilityd, Sessiond/PCEF, Pipelined, PolicyDB, Subscriberdb, OVS-data path, Enodebd, and the Control Proxy. Together, these components help to facilitate and manage data both to and from the user. 
```

Comment:
S-PGW is not a separate service. Let's remove it. An explanation like this would be useful: `MME service in Magma encapsulates MME functionality and parts of SPGW control plane features in LTE EPC. This name will be deprecated and changed to Accessd as its main purpose is to normalize the control signaling with the 4G/5G RAN nodes.`

Reply:
Thanks @ulaskozat! I will add that in.